Source/WebCore/rendering/HitTestResult.cpp

```diff
@@ -219,6 +220,32 @@ bool HitTestResult::isSelected() const
     return frame->selection().contains(m_hitTestLocation.point());
 }
 
+bool HitTestResult::allowsFollowingLink() const
+{
+    auto linkURL = absoluteLinkURL();
+    if (linkURL.isEmpty())
+        return false;
+
+    auto* innerFrame = innerNodeFrame();
```

Comment:
auto* should be RefPtr

Reply:
That code is similar to HitTestResult::hasEntireImage. Should we modify ::hasEntireImage to also use a RefPtr?